src/finalisers/umd.ts

```diff
@@ -117,6 +117,7 @@ export default function umd (
 	}
 
 	const wrapperIntro = `(function (global, factory) {
+			if(global === undefined && window !== undefined) global = window;
```

Comment:
This code throws if `global` is `undefined` due to the unchecked variable access (just verified it in Chrome 64). The assignment also needs to be changed as assignments to non-existing global variables fail in strict mode. This is what it should look like:
```javascript
if (typeof global === 'undefined' && typeof window !== undefined) window.global = window.
```